<launch>

  <arg name="uid" />
  <arg name="email" />
  <arg name="name" />

  <!-- launch the experiment controller node -->
  <node name="experiment_controller" pkg="bwi_exp1" type="controller.py" output="screen">
    <param name="graph_file" value="$(find bwi_exp1)/maps/graph.yaml" />
    <param name="experiment_file" 
           value="$(find bwi_exp1_solver)/config/experiment.yaml" />
    <param name="user_file" value="$(find bwi_exp1_solver)/data/users.yaml" />
    <param name="data_dir" value="$(find bwi_exp1_solver)/data" />
    <param name="uid" value="$(arg uid)"/>
    <param name="email" value="$(arg email)"/>
    <param name="name" value="$(arg name)"/>
  </node>

  <node name="robot_positioner" pkg="bwi_exp1_solver" 
        type="heuristic_robot_positioner">
    <param name="map_file" value="$(find bwi_exp1)/maps/exp1.yaml" />
    <param name="graph_file" value="$(find bwi_exp1)/maps/graph.yaml" />
    <param name="experiment_file" 
           value="$(find bwi_exp1_solver)/config/experiment.yaml" />
    <param name="default_robot_file" 
           value="$(find bwi_exp1)/config/default_robots.yaml" />
    <param name="robot_radius" value="0.25" />
    <param name="robot_padding" value="0.1" />
    <param name="search_distance" value="0.45" />
    <!-- <param name="debug" value="true" /> -->
    <!-- <param name="policy_file" value="$(find bwi_exp1)/policy.txt" /> -->
    <param name="goal_idx" value="46" />
  </node>

</launch>
